jmeter/plans/apply.rb

```diff
@@ -0,0 +1,92 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
+
+def url(path)
+  BASEURL + path
+end
+
+# 975 concurrent users, ~7 minutes per session
+test do
+  cookies clear_each_iteration: true
+  thread_count = 975
+
+  threads count: thread_count, continue_forever: true, duration: 420 do
```

Comment:
This duration was a bit of an afterthought, and was supposed to be indicative of the 7 minute average session time.  On a second look though, I don't think session duration is necessarily all that useful to try and model in this test. The session time allows us to estimate concurrent user count, and the test should probably just reflect that many users hitting the key paths in the app on a loop for whatever duration we think gives us a useful set of results (anywhere from 15-60 minutes?). WDYT?

Reply:
Yes, the approach we've taken for Manage/Vendor API tests is set the duration to 3600 and add think times in various threads. May be easier to manage this way.